plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/sql/SQLUtils.java

```diff
@@ -390,6 +388,16 @@ public static String trimQueryStatement(SQLSyntaxManager syntaxManager, String s
         return sql + trailingSpaces;
     }
 
+    private static boolean containsWhitespaces(@NotNull CharSequence charSequence) {
+        for (int i = 0; i < charSequence.length(); i++) {
+            char c = charSequence.charAt(i);
+            if (Character.isWhitespace(c)) {
+                return false;
+            }
+        }
+        return true;
+    }
+
```

Comment:
IMO it could be replaced with one-liner: `afterEnd.chars().noneMatch(Character::isWhitespace)`

Reply:
Fair point.